### PR TITLE
[WIP] Ticket revamp

### PR DESCRIPTION
**Pull request by Kmc2000**
## About The Pull Request

!!PROOF OF CONCEPT ONLY!!
Just checking if people like these, then I'm gonna PR them upstream and pull from bee again.

## Why It's Good For The Game

Makes our lives easier by implementing tickets that aren't totally awful.
Removes the godawful stat() based tickets panel 🤮 

-Mentors can only see mentor tickets
-Admins can see all tickets

## Changelog
:cl: Kmc2000
add: Adds ticket claiming! You can now state your intention to answer a ticket by claiming it, or relinquishing the claim if you change your mind. Other admins can still reply to the ticket, but it's marked as yours.
tweak: Admin tickets no longer show up on the status pane.  They have been moved to the show tickets verb
tweak: Mentorhelps have been entirely converted into tickets, and now inhabit the admin ticket list under a special category
/:cl:

### CODE REVIEW

**Comment by WhyIsCaeciliusTaken** [comment]
YES

PLEASE

GIVE US THIS UPSTREAM PLEASE.

**Comment by WhyIsCaeciliusTaken** [comment]
Two other things:
1: Where is the mark as IC button?
2: How do you ensure that mentors cannot see the PP, VV and SM buttons?

**Comment by Kmc2000** [comment]
> Two other things:
> 1: Where is the mark as IC button?
> 2: How do you ensure that mentors cannot see the PP, VV and SM buttons?

1: Under "re-classify"
2: They can see the options, but on clicking them they just get a "you dont' have permissions to do this"

File: config/admin_ranks.txt [hunk 1]
```diff
@@ -39,6 +39,11 @@ Include =
 Exclude = AUTOADMIN
 Edit =
 
+Name = Mentor
```
Comment (francinum) [inline config/admin_ranks.txt:42]: I assume this is in progress, Ideally mentors won't need `R_ADMIN`

File: nsv13/code/modules/admin/tickets.dm [hunk 2]
```diff
@@ -0,0 +1,194 @@
...
+	switch(action)
+		if("view")
+			viewing = AH
+			screen = "viewTicket"
+			return
+		if("back")
+			viewing = null
+			screen = "home"
+			return
+		if("re-class")
```
Comment (Jalleo) [inline nsv13/code/modules/admin/tickets.dm:149]: Personally I suggest a IC button seperate from here because those are probably more often for admins.

**Comment by francinum** [comment]
This will likely be outmoded by an upstream pr post #628

**Pull request closed without merging**